src/Data/Parameterized/TH/GADT.hs

```diff
@@ -25,6 +25,10 @@ module Data.Parameterized.TH.GADT
   , structuralHash
   , structuralHashWithSalt
   , PolyEq(..)
+    -- * Repr generators ("singletons")
```

Comment:
Haddock will treat anything between double quotes as a module name, which isn't what you want here. Do this instead:

```suggestion
    -- * Repr generators (\"singletons\")
```